profile_library/homewizard/HWE-SKT/model.json

```diff
@@ -0,0 +1,21 @@
+{
+  "aliases": [
+    "HWE-SKT",
```

Comment:
```suggestion
```

Already same as model directory, so doesn't have to be added as an alias.

Reply:
This is odd behavior IMHO. It makes meta data files incomplete.

../Frenck